### PR TITLE
Replace stub with hints and code solution

### PR DESCRIPTION
**Pull request by ailyntang**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/certifications/apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model/index.md [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+You need to do the following:
+1. Create a model of a person, using the schema from exercise 2
+2. Create a new person, including their attributes
+3. Save the new person you created
+4. Put your new person inside the `createAndSavePerson` function
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+## Solutions
+
+<details><summary>Solution 1 for `myApp.js`</summary>
```
Comment (RandellDawson) [inline guide/english/certifications/apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model/index.md]:
```suggestion
<details><summary>Solution #1 (Click to Show/Hide)</summary>
Code for `myApp.js`
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my suggested change.

**Comment by ailyntang** [comment]
> See my suggested change.

Thanks, I've committed. I'll do that in the future. Cheers

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**IMPORTANT INFORMATION!!!**

@ailyntang Until further notice, we are not going to be accepting/merging any new PRs for changes to the Guide articles in the certifications folder. There will be an announcement at a later date on the forum about how to update the curriculum related articles, but for now, just hold on to your solutions. We will work on approving the ones you have already created though.

Thank you.

**Pull request merged**